CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# The Odin Project Curriculum Contributing Guide
+
+Thank you for expressing interest in contributing to The Odin Project (TOP) curriculum! Before continuing through this guide, be sure you've read our [general contributing guide](https://github.com/TheOdinProject/.github/blob/ac0adb6bd151dca53798776783c5604d31e3389e/CONTRIBUTING.md), as it contains information that is important for all of our repos.
+
+This contributing guide assumes you have followed the instructions in our general contributing guide to fork and clone our curriculum repo.
+
+## Table of Contents
+- [How to Contribute](#how-to-contribute)
+- [Curriculum Linting](#curriculum-linting)
+
+## How to Contribute
+
+There are 2 main ways you can contribute to our curriculum:
+
+1. If you're new to contributing to open-source, only need to edit 1 file, or if you just want to make a really quick pull request (PR), you can click the "Improve this lesson on GitHub" link found at the end of each lesson. This will open the lesson file in its "edit" mode, allowing you to make any edits and submit a PR all through GitHub.
+2. If you're more experienced with contributing or need to edit more than 1 file, you can follow our instructions on [setting up a local clone](https://github.com/TheOdinProject/.github/blob/ac0adb6bd151dca53798776783c5604d31e3389e/CONTRIBUTING.md#setting-up-your-local-clone) from our general contributing guide. You should also read the sections that follow on how to open a PR.
+
+Regardless of the way you choose to open a PR, while working on an existing or a new lesson you **must** follow our [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) to ensure the layout and formatting is consistent across our curriculum.
+
+Before submitting a PR for an existing or a new lesson, you must also use our [Lesson Preview Tool](https://www.theodinproject.com/lessons/preview) to ensure the lesson Markdown is correctly formatted and rendering properly.
+
+## Curriculum Linting
+
+To help enforce the layout specified in our layout style guide, we use [markdownlint](https://github.com/DavidAnson/markdownlint). Whenever a PR is opened or has updates made to it, a workflow will run to check any files changed in the PR against common rules as well as custom rules specific to TOP. To make the workflow easier, we also strongly suggest that users who have a local clone run this linter locally before committing any changes. There are 2 ways you can do so:
+
+1. Install the [Markdownlint VSCode Plugin](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint). This plugin will automatically pickup our main configuration and flag issues with a squigly underline. Note that not the plugin will not currently flag issues with the structure of lessons.
```

Comment:
```suggestion
1. Install the [Markdownlint VSCode Plugin](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint). This plugin will automatically pick up our main configuration and flag issues with a squiggly underline. Note that the plugin will not currently flag issues with the structure of lessons or projects.
```

> Note that the plugin will not currently flag issues with the structure of lessons or projects.

I feel this could be a bit clearer as to what "the structure of lessons or projects" means? e.g. also mentioning it's the TOP004 rule, and/or that it's because the extension doesn't use the rules from the `(lesson|project).-*` configs